### ISSUE TITLE
Where can I find  "eval_ml" in train.py?

### ISSUE THREAD

**Issue by hoonsyang** [abhishekaich27/GAMA-pytorch#2]
`from eval_ml import evaluate_ml`

**Comment by abhishekaich27** [comment]
Updated the `train.py` file. Can you update on your end and check?

**Comment by hoonsyang** [comment]
Thanks it works.
Is it possible to utilize GAMA for training on single-object scene datasets such as ImageNet?

**Comment by abhishekaich27** [comment]
Definitely. ImageNet is a special case of multi-object scene (images mostly have one object). 

Just replace the victim models with suitable ones, adjust the loss functions for single labels and it should work fine.
